vllm/model_executor/tensorizer_loader.py

```diff
@@ -0,0 +1,294 @@
+import argparse
+import dataclasses
+import io
+import os
+import time
+import typing
+import warnings
+from dataclasses import dataclass
+from typing import Optional, Type, Union
+
+from tensorizer import DecryptionParams, TensorDeserializer, stream_io
+from tensorizer.utils import convert_bytes, get_mem_usage, no_init_or_tensor
```

Comment:
as tensorizer is optional, can we wrap all tensorizer imports in a try...except ImportError block and fail gracefully? we could re-raise in TensorizerAgent init

let's make sure that vllm can run without tensorizer installed!